ci/dictionary.txt

```diff
@@ -42,6 +42,7 @@ memoized
 MSRV
 natively
 onclick
+organisation
```

Comment:
Shouldn't this be "organization"?

Reply:
Depends – both _can_ be used. We should probably pick one spelling (I'll check what we're using elsewhere and update accordingly).